### PR TITLE
fix: remove on-prem specific docs

### PR DESCRIPTION
**Pull request by aptlin**
Removes sections about on-prem setup and mentions of backendUrl relevant only for on-prem users.

### CODE REVIEW

File: api/client/h-init.md [hunk 1]
```diff
@@ -21,12 +21,6 @@ If `projectId` is not set, then Highlight will not send any data. You can use th
 
 ## `HighlightOptions`
 
-### `backendUrl`* (string)*
-
-Specifies the URL that Highlight will send data to. You should not use this unless you are running an on-premise instance.
-
-You may be interested in [Proxying](/tips/proxying-highlight) to make sure your errors and sessions are not blocked by extensions.
```
Comment (Vadman97) [inline api/client/h-init.md:28]: the `backendUrl` option is still applicable for those using the `Proxying` feature.

File: api/client/h-init/index.md [hunk 2]
```diff
@@ -21,12 +21,6 @@ If `projectId` is not set, then Highlight will not send any data. You can use th
 
 ## `HighlightOptions`
 
-### `backendUrl`* (string)*
```
Comment (Vadman97) [inline api/client/h-init/index.md:24]: same here

File: api/client/h-toggle-session-feedback-modal.md [hunk 3]
```diff
@@ -21,12 +21,6 @@ If `projectId` is not set, then Highlight will not send any data. You can use th
 
 ## `HighlightOptions`
 
-### `backendUrl`* (string)*
```
Comment (Vadman97) [inline api/client/h-toggle-session-feedback-modal.md:24]: same here

**Review by Vadman97 — CHANGES_REQUESTED** [review]
we should keep the `backendUrl` references as customers that are proxying use that setting

File: api/client/h-init.md (hunk 1 above)
Comment (aptlin) [reply api/client/h-init.md]: My bad! Fixed in [6fbdb6d](https://github.com/highlight-run/docs/pull/3/commits/6fbdb6db971cf54393545dd8b521752ca5754622)

**Review by Vadman97 — APPROVED** [review]
thanks! 🚢 maybe we can set up some markdown auto-formatting / linting in this repo?

**Comment by aptlin** [comment]
Yes, and adding the same prettier config will make sure code snippets look the same

**Pull request merged**